libarchive/write.py

```diff
@@ -103,27 +147,34 @@ def write_cb_internal(archive_p, context, buffer_, length):
         ffi.write_set_bytes_in_last_block(archive_p, 1)
         ffi.write_set_bytes_per_block(archive_p, block_size)
         ffi.write_open(archive_p, None, open_cb, write_cb, close_cb)
-        yield ArchiveWrite(archive_p)
+        yield archive_obj_to_write(archive_p)
```

Comment:
`archive_obj_to_write` is a misleading name, please change it to something like `ArchiveWriter` or `ArchiveWriteClass` instead.